.gitlab/ci.yml

```diff
@@ -74,6 +74,7 @@ tarball suse-15:
         # build tools for CodeCompass
         - zypper install -y -t pattern devel_basis
         - zypper install -y binutils gcc-c++ gmp-devel
+        - zypper install -y groff groff-full
```

Comment:
Is `groff-base` not sufficient, like for Ubuntu?
(Also comment "*build tools for OpenLDAP*" missing here.)

Reply:
There is no supported groff-base package for SUSE 42 and 15 but groff is enough so I removed groff-full.